carla_host: "127.0.0.1"
carla_port: 2000
map_name: "Town06"
fixed_delta_seconds: 0.05
graphic_rendering: true
num_non_ego_vehicles: 999
simulation_duration: 300
state_estimation_frequency: 20
object_detection_frequency: 20
ego_init: 178
# ego_destination: 305 # left turn with red light
ego_destination: 94 # straight road with red light


agent_control_frequency: 10

ego_odom_topic: "ego_odom"
state_estimation_topic: "ego_state_estimation"
object_detection_topic: "object_detection"
motion_prediction_topic: "motion_prediction"

world_frame: "odom"
control_command_topic: "control_command"
show_hd_map_rviz: false